apps/meteor/packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -1850,6 +1851,8 @@
   "Enter_Normal": "Normal mode (send with Enter)",
   "Enter_to": "Enter to",
   "Enter_your_E2E_password": "Enter your E2E password",
+  "Enter_your_password_to_delete_your_account": "Enter your password to delete your account. This cannot be undone.",
+  "Enter_your_username_to_delete_your_account": "Enter your username to delete your account. This cannot be undone.",
```

Comment:
Remove translations that are not being used anymore... In this file and other languages files

Reply:
I'll do it for English, there's no need to do it for the other languages, deleting from EN will make the entry also be deleted from the other dictionaries

Reply:
For the record: "For_your_security_you_must_enter_your_current_password_to_continue" is in use in another component.